src/core/schema.ts

```diff
@@ -29,6 +30,22 @@ export const defaultSchema: Schema = {
     // [0-9]
     ...objectFromEntries("0123456789".split("").map((n) => [n, n])),
   },
+  defaultEventNames: {
```

Comment:
```suggestion
  defaultEventMappings: {
```

Since we use the term mappings already, might be nice to align with that naming?

Reply:
Obsoleted by https://github.com/hotwired/stimulus/pull/662/commits/2ddac5f652f5f42cfc7eeb1f912be6aefa8ef8e7

Reply:
I ended up dropping https://github.com/hotwired/stimulus/commit/2ddac5f652f5f42cfc7eeb1f912be6aefa8ef8e7 so this is relevant again. I'm open to renaming it.